Contributing.md

```diff
@@ -76,6 +83,36 @@ If you're reading this, you're probably creating a Pull Request or planning to d
 
 13. Create a PR to develop repository.
 
+# ERROR ❎ AND SOLUTION ✅ FOR COMMON PROBLEMS DURING SETUP 💻
+
+ 1. Please remember to include an .env file in both the frontend and backend directories. 
+ You can reference any two of the environment variables present in Discord for your configuration.
+
+ 2.Ensure Docker is running throughout the setup process. Make sure both the Redis stack and Progress 
+ components are up and running.
+
+ 3. To resolve port 3333 issues, use netstat to find the PID associated with it, then taskkill to terminate the process using that PID.
+   For Windows 
+
+     ```bash
+   "netstat -ano | findstr :3333"
+    "taskkill /PID <PID> /F"
+
+   ```
```

Comment:
code is not appearing in code-blocks. Try removing the line space.

Reply:
@ANIRUDHNARANG0 maybe write the corresponding linux command as well for above operation as well